cv_dv_utils/sv/beh_rand_delay/testbench/wave.do

```diff
@@ -0,0 +1,79 @@
+onerror {resume}
```

Comment:
This file is almost certainly tool-specific (by the look of it, Questasim).   Our strategy here is to place all tool-specific files into tool-specific directories.  For example, this could be in `lib/cv_dv_utils/sv/beh_rand_delay/testbench/tools/vsim/wave.do` or perhaps `lib/cv_dv_utils/sv/beh_rand_delay/tools/vsim/wave.do`.  This will, of course, affect the Makefiles.